build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     ext {
         opensearch_version = System.getProperty("opensearch.version", "3.0.0-SNAPSHOT")
         spring_version = "5.3.22"
-        jackson_version = "2.13.4"
+        jackson_version = "2.14.0"
```

Comment:
I don't think we should be checking this in.  It's unrelated content :)

Reply:
Build/GHA fail without the change. Will be rebased when https://github.com/opensearch-project/sql/pull/1066 merges.